docs(example): Add socket.io example (#1516)
remix-run/remix: examples/socket.io/README.md

# Express + Socket.io

Socket.io is the simples way to work with WebSockets in Node.js

This examples shows how to setup socket.io inside a Express server and together with a Remix app and how to connect from the browser.

> > Note: The code here needs the Express adapter and because of that it needs Node.js. This is not deployable to any platform not supporting a persistent running process like Serverless platforms.

Reviewer: ```suggestion
> Note: The code here needs the Express adapter and because of that it needs Node.js. This is only deployable to platforms supporting a persistent running process (unlike Serverless platforms).
```
